specification/components/schemas/Bundle.yaml

```diff
@@ -0,0 +1,32 @@
+type: object
+properties:
+  resourceType:
+    type: string
+    example: "Bundle"
```

Comment:
I think wherever value is constant, like `"resourceType": "Bundle"` this should be reflected in the schema as `default: "Bundle"` and not `example: "Bundle"`